### ISSUE TITLE
训练结果不理想

### ISSUE THREAD

**Issue by lhiceu** [xinzhuma/monodle#3]
首先感谢您优秀的工作。
我分别在 pytorch==1.1.0 torchvision==0.3.0 cudatoolkit=9.0和pytorch==1.7.1  torchvision==0.8.2  cudatoolkit==11.0.221环境下进行了训练，测试结果均未达到您提供的精度。请问是训练本身不稳定学要多次训练寻找最佳的模型？或者，您有其他更好的建议？

 pytorch==1.1.0 torchvision==0.3.0 cudatoolkit=9.0

'''
2021-06-23 11:42:27,283   INFO  Batch Size: 16
2021-06-23 11:42:27,283   INFO  Learning Rate: 0.001250
2021-06-23 19:24:40,438   INFO  ###################  Evaluation  ##################
2021-06-23 19:24:40,438   INFO  ==> Loading from checkpoint 'checkpoints/checkpoint_epoch_140.pth'
2021-06-23 19:24:40,570   INFO  ==> Done
2021-06-23 19:26:06,257   INFO  ==> Saving ...
2021-06-23 19:26:06,495   INFO  ==> Loading detections and GTs...
2021-06-23 19:26:06,978   INFO  ==> Evaluating (official) ...
2021-06-23 19:26:14,440   INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:90.5575, 89.1146, 80.3052
bev  AP:23.9675, 20.0978, 18.9213
3d   AP:17.7182, 14.8965, 14.3598
aos  AP:89.40, 87.96, 78.59
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:96.3973, 92.6146, 85.1240
bev  AP:21.1186, 17.1384, 14.8445
3d   AP:14.8023, 11.6153, 10.3170
aos  AP:95.11, 91.37, 83.15
Car AP@0.70, 0.50, 0.50:
bbox AP:90.5575, 89.1146, 80.3052
bev  AP:56.6758, 43.5572, 41.8785
3d   AP:53.0923, 41.2763, 39.3799
aos  AP:89.40, 87.96, 78.59
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:96.3973, 92.6146, 85.1240
bev  AP:55.9774, 43.5528, 39.3750
3d   AP:51.1366, 39.9114, 35.9619
aos  AP:95.11, 91.37, 83.15
'''

pytorch==1.7.1  torchvision==0.8.2  cudatoolkit==11.0.221

'''
2021-06-22 15:04:55,191   INFO  ###################  Training  ##################
2021-06-22 15:04:55,191   INFO  Batch Size: 16
2021-06-22 15:04:55,191   INFO  Learning Rate: 0.001250
2021-06-22 22:37:24,182   INFO  ###################  Evaluation  ##################
2021-06-22 22:37:24,183   INFO  ==> Loading from checkpoint 'checkpoints/checkpoint_epoch_140.pth'
2021-06-22 22:37:24,340   INFO  ==> Done
2021-06-22 22:39:02,019   INFO  ==> Saving ...
2021-06-22 22:39:02,249   INFO  ==> Loading detections and GTs...
2021-06-22 22:39:02,731   INFO  ==> Evaluating (official) ...
2021-06-22 22:39:09,084   INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:97.6635, 89.2212, 80.3331
bev  AP:23.0584, 19.6427, 18.4581
3d   AP:18.7181, 15.8292, 13.7209
aos  AP:96.85, 88.19, 78.86
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:98.4531, 92.7757, 85.1891
bev  AP:21.5541, 17.6774, 16.1178
3d   AP:16.0965, 12.8877, 11.0357
aos  AP:97.64, 91.60, 83.41
Car AP@0.70, 0.50, 0.50:
bbox AP:97.6635, 89.2212, 80.3331
bev  AP:58.4063, 44.6823, 42.8016
3d   AP:53.7804, 42.2338, 40.2918
aos  AP:96.85, 88.19, 78.86
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:98.4531, 92.7757, 85.1891
bev  AP:57.0261, 44.2483, 40.0147
3d   AP:51.1762, 40.1740, 36.1641
aos  AP:97.64, 91.60, 83.41
'''

**Comment by DuZzzs** [comment]
@lhiceu 你好，你是单卡训练的吗？

**Comment by lhiceu** [comment]
> @lhiceu 你好，你是单卡训练的吗？

不是，双卡训练，且没有改配置文件

**Comment by DuZzzs** [comment]
@lhiceu 方便告诉一下双卡训练的命令吗？我用train.sh多卡跑会卡住。非常感谢。

**Comment by xinzhuma** [comment]
code has been updated. you can try it again. @lhiceu
